src/com/hms/flexyosisoftconnector/OSIsoftConfig.java

```diff
@@ -160,6 +166,12 @@ public static void initConfig(String configFile) throws JSONException, IOExcepti
       }
     }
 
+    if (appConfig.has("httpTimeoutSeconds")) {
+      httpTimeoutSeconds = appConfig.getInt("httpTimeoutSeconds");

```

Comment:
Can you add a log message that the timeout is coming from the config and what its being set to?